test/sqs/sqs_e2e_test.go

```diff
@@ -0,0 +1,130 @@
+package sqs
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sqs"
+	"github.com/numaproj-contrib/numaflow-utils-go/testing/fixtures"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+	"log"
+	"testing"
+	"time"
+)
+
+type SqsSourceSuite struct {
+	fixtures.E2ESuite
+}
+
+const (
+	AWS_ACCESS_KEY = "access-key"
+	AWS_REGION     = "us-east-1"
+	AWS_SECRET     = "access-secret"
+	AWS_QUEUE      = "numaflow-test"
+	AWS_ENDPOINT   = "http://127.0.0.1:5000"
+)
+
+func setupQueue(client *sqs.SQS, queueName string) (*string, error) {
+	params := &sqs.CreateQueueInput{
+		QueueName: aws.String(queueName),
+	}
+
+	response, err := client.CreateQueue(params)
+	if err != nil {
+		fmt.Println("Error creating queue:", err)
+		return nil, err
+	}
+	return response.QueueUrl, nil
+}
+
+func SendMessage(sqsClient *sqs.SQS, queueUrl string, messageBody string) error {
+	_, err := sqsClient.SendMessage(&sqs.SendMessageInput{
+		QueueUrl:    &queueUrl,
+		MessageBody: aws.String(messageBody),
+	})
+	return err
+}
+
+func CreateAWSSession(accessKey, region, secret, endPoint string) *session.Session {
+	sess := session.Must(session.NewSessionWithOptions(session.Options{
+		Config: aws.Config{
+			Region:      aws.String(region),
+			Credentials: credentials.NewStaticCredentials(accessKey, secret, ""),
+			Endpoint:    aws.String(endPoint),
+		},
+		SharedConfigState: session.SharedConfigDisable,
+	}))
+	return sess
+}
+
+func (suite *SqsSourceSuite) SetupTest() {
+
+	suite.T().Log("e2e Api resources are ready")
+	time.Sleep(10 * time.Second)
```

Comment:
we can check the status of resource to be ready maybe in follow up PR

Reply:
removed ,not required as such